src/System.Management.Automation/engine/remoting/commands/EnterPSHostProcessCommand.cs

```diff
@@ -565,14 +574,45 @@ internal static IReadOnlyCollection<PSHostProcessInfo> GetAppDomainNamesFromProc
                                     }
                                 }
```

Comment:
This would indicate a malformed named pipe file.  I think it would still be useful to show it, it may indicate malicious use.

Reply:
With the current implementation, we won't be showing it because `id = -1` in that case, and it will cause an exception when calling `Process.GetProcessById(id)` in the `if (id != currentPid)` block.

Reply:
We could emit a verbose message and continue